src/raftstore/store/snap.rs

```diff
@@ -109,11 +235,11 @@ impl SnapFile {
     }
 
     pub fn init(&mut self) -> io::Result<()> {
-        if self.exists() || self.tmp_file.is_some() {
+        if SnapshotFileWriter::exists(self) || self.tmp_file.is_some() {
```

Comment:
There are two `exists()` in two trait. The compiler will report error if using `self.exists()` without specifying trait name.